src/components/common/TmBalance.vue

```diff
@@ -358,6 +358,7 @@ export default {
           )
           this.sentToGA = true
         }
+        // sending to ga only once
```

Comment:
1. are we still sending to GA?
2. is this comment in the right place? shouldn't it be on line 359?

Reply:
1. Why not? )
2. Ooops

We can do it in API only, but without browser-id